src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -432,7 +432,11 @@ const SingleActiveFIMap = (props: MapSingleFIProps & RouteComponentProps<RoutePa
                         ({goalReport.prettyPercentAchieved})
                       </p>
                       <br />
-                      <ProgressBar value={goalReport.percentAchieved} max={1} />
+                      <ProgressBar
+                        value={0.012}
```

Comment:
Looks like this is left over from testing?  We should revert back to `value={goalReport.percentAchieved}`

Reply:
Yeah I have reverted back in the next commit